src/backend/src/utils/teams.utils.ts

```diff
@@ -72,3 +72,15 @@ export const getTeamsFromUsers = (users: UserWithTeams[]): Team[][] => {
     return teams;
   });
 };
+
+/**
+ * Removes all users in the second list from the first list. Returns the userIds of
+ * all users in the first list filtered to exclude those users in the second list.
+ * @param currentUsers The primary list of users
+ * @param usersToRemove the list of users to remove from currentUsers
+ * @returns the userIds of all users in currentUsers that aren't in usersToRemove
+ */
+export const removeUsersFromList = (currentUsers: User[], usersToRemove: User[]): number[] => {
```

Comment:
You could make this return type of `User[]` to avoid having to define another user utils function like this:
```
const userIdsToRemove = usersToRemove.map((user) => user.userId);
return currentUsers.filter((user) => !userIdsToRemove.includes(user.userId))
```